playbooks/roles/appsembler_reporting/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+appsembler_reporting_log_dir: /edx/var/log/appsembler_reporting
+appsembler_reporting_scripts_dir: "{{ edxapp_app_dir }}/appsembler_reporting"
+appsembler_reporting_jobs:
+  grades:
+    cron_h: 0
+    cron_m: 0
+  courses:
+    cron_h: 1
+    cron_m: 0
```

Comment:
Please favor the following format as it's more Ansible-ic:

```
appsembler_reporting_jobs:
  - name: grades
    cron_h: 0
    cron_m: 0
  - name: courses
    cron_h: 1
    cron_m: 0
```

So I don't have to wonder what `item.value` means.